commit 54d5b52a2ee926b5c95ec1ba52f7b135ec1e89e1
Merge: 9040446 0ce252b
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Fri Jan 23 08:36:25 2015 -0800

    Merge pull request #1399 from beanz/use-bed-defines-not-extruder-ones
    
    Code wont compile with extruder but without bed runaway protection defined.

diff --git a/Marlin/temperature.cpp b/Marlin/temperature.cpp
index f6eadaa..b83fdd7 100644
--- a/Marlin/temperature.cpp
+++ b/Marlin/temperature.cpp
@@ -572,7 +572,7 @@ void manage_heater()
 
   #if TEMP_SENSOR_BED != 0
   
-    #ifdef THERMAL_RUNAWAY_PROTECTION_PERIOD && THERMAL_RUNAWAY_PROTECTION_PERIOD > 0
+    #if defined(THERMAL_RUNAWAY_PROTECTION_BED_PERIOD) && THERMAL_RUNAWAY_PROTECTION_BED_PERIOD > 0
       thermal_runaway_protection(&thermal_runaway_bed_state_machine, &thermal_runaway_bed_timer, current_temperature_bed, target_temperature_bed, 9, THERMAL_RUNAWAY_PROTECTION_BED_PERIOD, THERMAL_RUNAWAY_PROTECTION_BED_HYSTERESIS);
     #endif
 
